### DISCUSSION TITLE
Radarr doesn't update

### DISCUSSION THREAD

**Discussion by uberguy16** [tteck/Proxmox#3427] [Help Needed] [answered]
I haven't noticed this with any of the other containers, but the Radarr LXC doesn't seem to update.  I'm still running v5.4.6.8723 yet they are up to v5.8.3.8933. When I run update, it shows Updated in a second yet never does actually update.

**Comment by tteck** [comment, marked as answer]
There is currently no update path available.

**Comment by jfamiens** [comment]
@tteck I have the same issue, should i understand that there is simply no update available ?

**Comment by oOStroudyOo** [comment]
On Radarr > Settings > General > Show Advanced > Automatic Updates > Tick

Nothing to do with the scripts. The Arrs have built in update's.

**Reply by uberguy16** [reply]
Well, that was easy. :)  I've been running Radarr in Docker for years so used to having to update the container. Thanks.

**Comment by chenks** [comment]
yeah that keeps radarr updated, but what about keeping the actual LXC updated?
strange that Sonarr would have an update path but Radarr wouldn't

**Reply by oOStroudyOo** [reply]
You can run a Cron job for apt update.

My personally, I run this and changed it to automatically run daily at midnight: https://tteck.github.io/Proxmox/#proxmox-ve-cron-lxc-updater

**Reply by chenks** [reply]
> You can run a Cron job for apt update.
> 
> My personally, I run this and changed it to automatically run daily at midnight: https://tteck.github.io/Proxmox/#proxmox-ve-cron-lxc-updater

how do you get it to run in unattended mode?
at the start you have to manually include/exclude the LXCs before it runs.

**Reply by oOStroudyOo** [reply]
Not really sure what you mean? I'm no Linux expert but aren't all Cron jobs unattended?

I've just tried the Tteck script and it works fine, this is all I did:
Open Shell for your Node
Run `bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/misc/cron-update-lxcs.sh)"`
Select `add schedule`
`y` to confirm

**To include/exclude LXCs:**
Open Shell for your Node
Type `crontab -e`
Add the container IDs that you want to exclude by putting `-s your container ids`  between  `0 0 * * 0 PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin /bin/bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/misc/update-lxcs-cron.sh)"` and `>>/var/log/update-lxcs-cron.log 2>/dev/null`

You can also change when it runs by editing the Cron expression `0 0 * * 0`
If you're not familiar, this is a good expression generator https://crontab.cronhub.io/

**Examples**

_This updates all LXCs at midnight every Sunday, excluding containers 103 and 111:_
`0 0 * * 0 PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin /bin/bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/misc/update-lxcs-cron.sh)" -s 103 111 >>/var/log/update-lxcs-cron.log 2>/dev/null`

_This updates all LXCs at 8am every day:_
`0 8 * * * PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin /bin/bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/misc/update-lxcs-cron.sh)" >>/var/log/update-lxcs-cron.log 2>/dev/null`

_This updates all LXCs at 6pm on the 1st of every month, excluding containers 109 110 111 and 112:_
`0 18 1 * * PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin /bin/bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/misc/update-lxcs-cron.sh)" -s 109 110 111 112 >>/var/log/update-lxcs-cron.log 2>/dev/null`

**Reply by oOStroudyOo** [reply]
You might have been getting these 2 Tteck scripts mixed up @chenks 

https://tteck.github.io/Proxmox/#proxmox-ve-cron-lxc-updater
https://tteck.github.io/Proxmox/#proxmox-ve-lxc-updater

**Reply by chenks** [reply]
yes i hadn't noticed there was a cron version of the script also
